src/EA.Weee.Web.Tests.Unit/Areas/Admin/Controllers/ManageEvidenceNotesControllerTests.cs

```diff
@@ -20,6 +21,16 @@
 
     public class ManageEvidenceNotesControllerTests : ManageEvidenceNotesControllerTestsBase
     {
+        private readonly DateTime currentDate;
+
+        public ManageEvidenceNotesControllerTests()
+        {
+            currentDate = new DateTime(2019, 1, 1);
+            A.CallTo(() => WeeeClient.SendAsync(A<string>._, A<GetApiUtcDate>._)).Returns(currentDate);
+
+            A.CallTo(() => SessionService.GetTransferSessionObject<object>(ManageEvidenceController.Session, SessionKeyConstant.AASelectedComplianceYear)).Returns(null);
```

Comment:
I've changed the way that the compliance year is returned to the manage screens when viewing notes now so we dont need this session. I'll be removing the one from the AATF manage evidence notes index action.

Reply:
okay, is that change in dev?

Reply:
Yeah now when returning from viewing of notes it should use the the note compliance year to pass that back to the manage evidence controller. 

for example
 Url.Action("Index", "ManageEvidenceNotes",
            new { @pcsId = Model.SchemeId, @tab = Model.RedirectTab, @selectedComplianceYear = Model.ComplianceYear }), "Manage evidence notes")).... so when the new view screens are created we just need to use the current note compliance year rather than trying to remember the selection... 

I'll be making the change in the AATF area soon...

Reply:
okay, i will put it.